src/logcat/parser.coffee

```diff
@@ -1,8 +1,10 @@
 {EventEmitter} = require 'events'
+parser = require "./parser/binary"
 
 class Parser extends EventEmitter
   @get: (type) ->
-    parser = require "./parser/#{type}"
+    if type !== 'binary'
+      throw new Error "Unknown parser type #{type}"
```

Comment:
This would be more CoffeeScripty with an `unless` following the throw. But we're done with CS anyway.